wunderio/client-UK-FSA-beta
# FSA-1004: Change font on 'print this page' button

To test:
- Go to any article/content
- The content toolbar (view pdf/print/share) should now all have the same font.
